server/game/cards/08.5-TFM/TheFaithMilitant.js

```diff
@@ -0,0 +1,43 @@
+const AgendaCard = require('../../agendacard.js');
+
+class TheFaithMilitant extends AgendaCard {
+    setupCardAbilities(ability) {
+
+        this.persistentEffect({
+            match: card =>
+                !card.hasTrait('The Seven') &&
+                ['character', 'location'].includes(card.getType()) &&
+                card.controller === this.controller,
+            effect: ability.effects.cannotGainPower()
+        });
+
+        this.reaction({
+            when: {
+                afterChallenge: event =>
+                    event.challenge.winner === this.controller &&
+                    event.challenge.loser.faction.power >= 1
+            },
+            cost: ability.costs.kneelFactionCard(),
+            target: {
+                cardCondition: card =>
+                    card.location === 'play area' &&
+                    card.controller === this.controller &&
+                    card.getType() === 'character' &&
+                    card.hasTrait('The Seven') &&
+                    this.game.currentChallenge.isParticipating(card)
+            },
+            handler: context => {
+                var otherPlayer = context.event.challenge.loser;
```

Comment:
var --> let

Reply:
as this was coming from copy/past from another card, I've reviewed "var" use in card implementation and filed #1785